docs/DeveloperGuide.md

```diff
@@ -1,3 +1,1140 @@
+> into an existing ArrayList\<Drug\> inventory which contains all the inventory
+> objects.
+
+Saving Drug objects into .txt file
+
+> The command loop then calls DrugStorage.writeToFile() which starts the
+> process of writing details of all existing Drug objects within the
+> ArrayList to the specified .txt file
+
+list \[Drug name\]:
+
+> When the user inputs this command, the input will be parsed into
+> Parser.java, which will call the DrugActions.listDrug() method in the
+> DrugActions.java.
+>
+> The most updated database of drugs saved will be listed.
+
+delete \[Drug name\]:
+
+> When the user inputs this command, the input will be parsed into
+> Parser.java, which will call the DrugActions.deleteDrug() method in
+> the DrugActions.java.
+>
+> Everything related to that specific inventory will be removed.
+
+help:
+
+> When the user inputs this command, the input will be parsed into
+> Parser.java, which will call the DrugUI.printHelpMessage() method in
+> the DrugUI.java.
+>
+> The available features, add, list, delete, help, and return will be
+>
+> displayed for the user.
+>
+> The program will wait for the user to input one of the features.
+
+return:
+
+> When the user inputs this command, the input will be parsed into
+> Parser.java, it will bring the user back to the main menu of the
+> program.
+
+ 
+
+**Documentation, logging testing, configuration, dev-ops**
+----------------------------------------------------------
+
+### **Product scope**
+
+**Target user profile:**
+
+-   Has a need to manage a huge amount of data like medical records,
+    > schedules and doctor info
+
+-   Prefers a no-frills functional app over other types
+
+-   Can type reasonably fast and accurate
+
+-   Does not need a mouse
+
+-   Reasonably comfortable using CLI applications
+
+**Value Proposition:** A hospital management system that is faster and
+minimal compared to a typical GUI application
+
+### User stories
+
+  **Priority**   **As a\...**   **I want to\...**                                       **So that I can\...**
+  -------------- -------------- ------------------------------------------------------- --------------------------------------------------
+  \* \* \*       new user       quickly refer to usage instructions                     quickly get on track with the workflow
+  \* \* \*       user           add a new staff/patient                                 
+  \* \* \*       user           delete staff/patients                                   remove entries i no longer need
+  \* \* \*       user           quickly look up schedules for both nurses and doctors   plan my schedule better
+  \* \* \*       user           quickly look up inventory inventories                        plan what and when to restock our inventory supplies
+  \*             user           have the program recognize slight errors in typing      have leeway working in a high-stress environment
+
+### 
```

Comment:
There seems to be an additional "###". It also appeared in a few other areas in the document, remember to remove it!